src/core/models/answersconfig.js

```diff
@@ -0,0 +1,147 @@
+/** @module AnswersConfig */
+import SearchConfig from '../models/searchconfig';
+import VerticalPagesConfig from '../models/verticalpagesconfig';
+
+/**
+ * AnswersConfig is a model that is used that is used to turn a raw configuration into a data model.
+ * It is used to initialize the Answers SDK
+ */
+export default class AnswersConfig {
+  constructor (rawConfig) {
+    /**
+     * Required, the Yext Answers API Key
+     * @type {string}
+     */
+    this.apiKey = rawConfig.apiKey;
+
+    /**
+     * The internal Yext environment
+     * @type {string}
+     */
+    this.environment = rawConfig.environment;
+
+    /**
+     * Required, the key used for your Answers Experience
+     * @type {string}
+     */
+    this.experienceKey = rawConfig.experienceKey;
+
+    /**
+     *  Optional, initialize components here, invoked when the Answers component library is loaded/ready.
+     *  If components are not added here, they can also be added when the init promise resolves
+     * @type {function()}
+     */
+    this.onReady = rawConfig.onReady;
+
+    /**
+     * Optional*, Yext businessId, *required to send analytics events
+     * @type {number|string|*}
```

Comment:
why do we allow more than one type here? i would think that is a number based on its usage in AnalyticsReporter. although looking at our documentation and jambo `ANSWERS.init` call, it looks like we pass in a string.

Reply:
I believe that was my auto-jsdoc. There is a string usage of it in tests/acceptance/fixtures/html/vertical.html where as other places have it being an integer